doc/content/examples/example9.md

```diff
@@ -0,0 +1,95 @@
+- A groove is made in the soil domain to accommodate the foundation using the `Webcut > Tool` option. Then, the common surfaces between the soil and the foundation are merged using the `merge all` command. Now, the foundation and the soil are in continuum.
+- The vertical structure is connected to the foundation through rigid horizontal beams so that while the structure's translation is constrained, its rotation remains unconstrained. It has to be made sure that the node at which the structure and horizontal beams intersect is not merged with the foundation. However, the other nodes connecting the horizontal rigid beams to the foundation should be merged with the foundation.
+- The vertical structure is represented by a beam of the required length.
+
+The soil domain is meshed such that each element is approximately a cube of side 2 $m$. While this results in a somewhat coarser mesh, the computational burden will still be manageable. The foundation is meshed such that it compatible with the soil meshing. Rigid foundation meshing should be less of a concern in relation to the flexible soil meshing. While the horizontal connector beams are meshed into single elements, the vertical beam is meshed into five elements. In total, the model has 21,685 elements with 24,358 nodes. This model is presented in the figure below.
+
+!media media/examples/ex09_Mesh.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Mesh
+       caption=Cubit model for the SSI analysis.
+
+## Modeling in MASTODON
+
+This example consists of three components: (i) Modeling the individual structural response; (ii) Modeling the individual soil response; and (iii) Modeling the Soil Structure Interaction. The components are now individually discussed.
+
+### Individual structural response
+
+The input file for simulating the individual structural response is presented below. The vertical beam representing the structure has a mass lumped at the top in addition to having a density. The horizontal beams connected to the rigid foundation are massless and they facilitate a rotational degree of freedom for the vertical beam at the node where it is intersecting with the foundation. The rigidity of both the vertical and the horizontal beams is achieved by setting properties such as area, moment of inertia, and Young's modulus to high values. The rigidity of the foundation is achieved by setting its Young's modulus to a high value. The Ormsby Wavelet was applied at the bottom of the rigid foundation and the response at the top of the structure was recorded. The figure below presents the 1% damped acceleration response spectra at the top of the structure. It is observed that at about 0.23s, the response spectra is peaking. This is the fundamental period of the structure. [!citet](BolisettiThesis) obtained a very similar value when modeling in LS-Dyna.
```

Comment:
"The figure below presents the 1% damped acceleration response spectra at the top of the structure. It is observed that at about 0.23s, the response spectra is peaking. This is the fundamental period of the structure. [!citet](BolisettiThesis) obtained a very similar value when modeling in LS-Dyna." >>>> Figure 3 shows the 1% damped acceleration response spectra at the structure top with the fundament period of about 0.23 s. This value shows a good agreement with that in the LS-Dyna model by Bolisetti (2014).